### ISSUE TITLE
a11y issue in collapsed fieldsets

### ISSUE THREAD

**Issue by btopro** [elmsln/issues#1268]
Respond to click events, and hover to show intent of a button, yet they are not buttons or tabbable. This is because a `div` was being made to behave like a button yet it lacked all of the actual capabilities of a button. When possible, always use the semantic type to represent the thing in question.

**Comment by btopro** [comment]
shoveling


shoveling


shoveling


shoveling


shoveling


shoveling
